Sources/MlemMiddleware/Content Models/Comment/AnyComment.swift

```diff
@@ -67,4 +67,13 @@ public extension AnyComment {
             upgradeOperation: upgradeOperation ?? { try await $0.upgrade() }
         )
     }
+    
+    func refresh(upgradeOperation: ((any Base) async throws -> any Base)?) async throws {
```

Comment:
Maybe nicer to make this a default and avoid the unwrap below?

```suggestion
    func refresh(upgradeOperation: ((any Base) async throws -> any Base) = { try await $0.upgrade() }) async throws {
```

Reply:
`Upgradable` requires implementation of the `refresh` method. If it were non-optional and we were calling `refresh` on `any Upgradable`, we'd have to write out the default value every time:

```swift
model.refresh { try await $0.upgrade() }

// Another example, where upgradeOperation is optional
model.refresh(upgradeOperation: upgradeOperation ?? { try await $0.upgrade() })
```

Having it accept `nil` makes the syntax a little simpler:

```swift
model.refresh(upgradeOperation: nil)

// Where upgradeOperation is optional
model.refresh(upgradeOperation: upgradeOperation)
```

Another benefit of having it accept `nil` is that if we ever need to change the default implementation, we could change it easily in one place without having to modify everywhere `{ try await $0.upgrade() }` is written.

Reply:
Fair enough 👍